elastic/apm-server
# add config reload debouncer

prevent multiple reloads in rapid succession.

to be tested in cloud, an attempt at seeing if this fixes the bug setting the output registry (https://github.com/elastic/apm-server/issues/8383)

## how to test

1. Start the stack in cloud
2. In the admin console, enable logs and metrics. Send them to this deployment.
3. Send events to the apm-server (grab the secret token from the apm integration, but do not update the integration!)
4. Navigate to stack monitoring; observe that metrics appear under "Output Events Rate"
